src/core/runner.ts

```diff
@@ -320,8 +325,8 @@ export default class Runner extends EventEmitter {
   registerJourney(journey: Journey, context: JourneyContext) {
     this.currentJourney = journey;
     const timestamp = getTimestamp();
-    const { params } = context;
-    this.emit('journey:start', { journey, timestamp, params });
+    const { params, networkConditions } = context;
+    this.emit('journey:start', { journey, timestamp, params, networkConditions });
```

Comment:
I don't think we need it for each journey as we dont allow users to modify it per journey, instead it should be only for top level event `start` which is for every run. https://github.com/elastic/synthetics/blob/a621b530a9559538c5d52eefbaabb8c9c9e62c8e/src/core/runner.ts#L437

and we write it as part of `synthetics/metadata` event.